rotkehlchen/externalapis/beaconchain.py

```diff
@@ -281,3 +277,60 @@ def get_eth1_address_validators(self, address: ChecksumEthAddress) -> List[Valid
                 f'Beaconchai.in eth1 response processing error. Missing key entry {str(e)}',
             ) from e
         return validators
+
+    def get_validator_deposits(
+            self,
+            indices_or_pubkeys: Union[List[int], List[Eth2PubKey]],
+    ) -> List[Eth2Deposit]:
+        """Get the deposits of all the validators given from the list of indices or pubkeys
+
+        Queries in chunks of 100 due to api limitations
+
+        May raise:
+        - RemoteError due to problems querying beaconcha.in API
+        """
+        chunks = list(get_chunks(indices_or_pubkeys, n=100))  # type: ignore
+        data = []
+        for chunk in chunks:
+            result = self._query(
+                module='validator',
+                endpoint='deposits',
+                encoded_args=','.join(str(x) for x in chunk),
```

Comment:
Didn't know that you could make the comprehension like this (without the () or [])